heudiconv/dicoms.py

```diff
@@ -194,6 +194,10 @@ def group_dicoms_into_seqinfos(files, file_filter, dcmfilter, grouping):
         else:
             sequence_name = 'Not found'
 
+        if dcminfo.get([0x18,0x86], None):
+            # Siemens scanners
+            echo = dcminfo[0x18,0x86].value
```

Comment:
yet to decide (ref: https://github.com/nipy/heudiconv/issues/162#issuecomment-385428448) either we should handle it manually at all or rely on annotation from dcm2niix.  That one would exist though only whenever `--bids` side car file is generated...  my points somewhat -- less we (re)implement of logic dcm2niix provides (and tests for) inside, the better

Reply:
I agree. if its trivial to implement we can do this approach. otherwise the multi-echo support will be implemented at a completely different stage of the heudiconv run.